### PR TITLE
fix sory key getter in SimpleColumn

### PR DESCRIPTION
**Pull request by nguyenk**

### CODE REVIEW

File: src/Mouf/Html/Widgets/EvoluGrid/SimpleColumn.php [hunk 1]
```diff
@@ -153,7 +153,7 @@ public function isSortable() : bool
      */
     public function getSortKey()
     {
-        return $this->key;
+        return $this->sortKey;
```
Comment (moufmouf) [inline src/Mouf/Html/Widgets/EvoluGrid/SimpleColumn.php]: Isn't it the key by default is the sort key is not provided?

Maybe something more like:

```suggestion
        return $this->sortKey ?: $this->key;
```

Comment (nguyenk) [reply src/Mouf/Html/Widgets/EvoluGrid/SimpleColumn.php]: Dont know, in TwigColumn, getter is the same :
```php
public function getSortKey()
    {
        return $this->sortKey;
    }
```

Comment (nguyenk) [reply src/Mouf/Html/Widgets/EvoluGrid/SimpleColumn.php]: ... but I can change it to ```return $this->sortKey ?: $this->key;```

Comment (moufmouf) [reply src/Mouf/Html/Widgets/EvoluGrid/SimpleColumn.php]: Because in TwigColum, there is no "$this->key", right?

**Pull request merged**